shared/editor/plugins/Mermaid.ts

```diff
@@ -66,6 +67,7 @@ function getNewState({
               htmlLabels: false,
             },
             fontFamily: "inherit",
+            theme: stores.ui.resolvedTheme === "light" ? "default" : "dark",
```

Comment:
Ah, thanks for trying. Unfortunately you cannot import things from `~/` into `shared` as that's importing client dependencies into the server. I think you can pass it to the plugin from `node.editor.props.theme`, you can listen for changes with `window.addEventListener("theme-changed"`

Reply:
I see that the same approach is used within `editor.components.ComponentView`. Also within this ComponentView I should have access to `node.editor.props.theme`?